.github/workflows/ci-test.yml

```diff
@@ -0,0 +1,42 @@
+# Github action definitions for unit-tests with PRs.
+
+name: tft-unit-tests
+on:
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+  workflow_dispatch:
+
+jobs:
+  unit-tests:
+    if: github.actor != 'copybara-service[bot]'
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+
+    strategy:
+      matrix:
+        python-version: ['3.9', '3.10', '3.11']
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ matrix.python-version }}
+        cache: 'pip'
+        cache-dependency-path: |
+          setup.py
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install --upgrade wheel setuptools
```

Comment:
Probably don't need the `pip` upgrade here. Do we need `wheel` and `setuptools` here?

Reply:
> Probably don't need the `pip` upgrade here.

I can remove the `pip` upgrade.

> Do we need `wheel` and `setuptools` here?

They were part of the original install instructions, but testing on my local machine seems to be safe without them

> One other question - can we do `pip install <whatever>` or does it need to be invoked by `python -m pip`?

I will change to just `pip`